app/uk/gov/hmrc/cdsreimbursementclaimfrontend/views/utils/CheckYourAnswersHelper.scala

```diff
@@ -20,17 +20,22 @@ import play.api.i18n.{Lang, Langs, MessagesApi}
 import uk.gov.hmrc.cdsreimbursementclaimfrontend.controllers.actions.RequestWithSessionData
 import uk.gov.hmrc.cdsreimbursementclaimfrontend.controllers.claims.routes
 import uk.gov.hmrc.cdsreimbursementclaimfrontend.controllers.supportingevidence.{routes => fileUploadRoutes}
-import uk.gov.hmrc.cdsreimbursementclaimfrontend.models.CompleteClaim
+import uk.gov.hmrc.cdsreimbursementclaimfrontend.models.{CompleteClaim}
 import uk.gov.hmrc.cdsreimbursementclaimfrontend.models.finance.MoneyUtils.formatAmountOfMoneyWithPoundSign
 import uk.gov.hmrc.cdsreimbursementclaimfrontend.models.ids.{EntryNumber, MRN}
 import uk.gov.hmrc.cdsreimbursementclaimfrontend.models.upscan.SupportingEvidence
+import uk.gov.hmrc.cdsreimbursementclaimfrontend.services.FeatureSwitchService
 import uk.gov.hmrc.govukfrontend.views.viewmodels.content.Text
 import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.{SummaryListRow, _}
 
 import javax.inject.{Inject, Singleton}
 
 @Singleton
-class CheckYourAnswersHelper @Inject() (implicit langs: Langs, messages: MessagesApi) {
+class CheckYourAnswersHelper @Inject() (implicit
+  langs: Langs,
+  messages: MessagesApi,
```

Comment:
the langs implicit needs to be removed and the MessagesApi needs to be replaced with Messages as discussed